### PR TITLE
FIX: add missing canView check in json

### PR DESCRIPTION
**Pull request by andreaspiening**
Currently the JSON formatted DataLists show non-existent hasOnes and related objects, that the user !canView.

### CODE REVIEW

File: src/DataFormatter/JSONDataFormatter.php [hunk 1]
```diff
@@ -140,6 +143,7 @@ public function convertDataObjectToJSONObject(DataObjectInterface $obj, $fields
                 $innerParts = array();
                 $items = $obj->$relName();
                 foreach ($items as $item) {
+                    if (!$item->canView()) continue;
```
Comment (robbieaverill) [inline src/DataFormatter/JSONDataFormatter.php]: Could you please format this for PSR-2?

**Comment by andreaspiening** [comment]
sorry, sure. old habits die hard

**Comment by andreaspiening** [comment]
Cheers. It's a pleasure working with you !

**Comment by robbieaverill** [comment]
No problem, thanks for contributing!

**Pull request merged**